.gitignore

```diff
@@ -26,3 +26,6 @@ Misc/TypeMap
 /dat
 /doc
 /*.kdev4
+
+/build/*
```

Comment:
I usually don't add `/build` here any more, since it's a generic name that isn't created by any specific automatic build system (that I know of), and most will instead generate several build directories with different names.  Instead, you can generate your cmake build out-of-source, or create a .gitignore file in the build directory with only a `*` line as content.